tests/test_file_handler.py

```
@@ -0,0 +1,49 @@
+
+
+def test_file_handler_writes_to_file(tmp_path: Path) -> None:
+    path = tmp_path / "out.log"
+    handler = FemtoFileHandler(str(path))
+    handler.handle("core", "INFO", "hello")
+    del handler
+    gc.collect()
+    assert path.read_text() == "core [INFO] hello\n"
+
+
+def test_file_handler_multiple_records(tmp_path: Path) -> None:
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)